docs/_docs/01_getting-started/quick-start.md

```diff
@@ -59,91 +62,91 @@ The basic usage pattern for writing automated tests with Terratest is to:
 To make this sort of testing easier, Terratest provides a variety of helper functions and patterns for common infrastructure testing tasks, such as testing Terraform code, testing Packer templates, testing Docker images, executing commands on servers over SSH, making HTTP requests, working with AWS APIs, and so on.
 
 
-## Example #1: Terraform
-Let’s say you have the following (simplified) [Terraform](https://www.terraform.io/) code to deploy a web server in AWS (if you’re new to Terraform, check out our [Comprehensive Guide to Terraform](https://blog.gruntwork.io/a-comprehensive-guide-to-terraform-b3d32832baca)):
-
-```hcl
-provider "aws" {
-  region = "us-east-1"
-}
-
-resource "aws_instance" "web_server" {
-  ami                    = "ami-43a15f3e" # Ubuntu 16.04
-  instance_type          = "t2.micro"
-  vpc_security_group_ids = ["${aws_security_group.web_server.id}"]
-  # Run a "Hello, World" web server on port 8080
-  user_data = <<-EOF
-              #!/bin/bash
-              echo "Hello, World" > index.html
-              nohup busybox httpd -f -p 8080 &
-              EOF  
-}
-
-# Allow the web app to receive requests on port 8080
-resource "aws_security_group" "web_server" {
-  ingress {
-    from_port   = 8080
-    to_port     = 8080
-    protocol    = "tcp"
-    cidr_blocks = ["0.0.0.0/0"]
-  }
-}
-
-output "url" {
-  value = "http://${aws_instance.web_server.public_ip}:8080"
-}
+## Example #1: Terraform "Hello, World"
+
+Let's start with the simplest possible [Terraform](https://www.terraform.io/) code, which just outputs the text, 
+"Hello, World" (if you’re new to Terraform, check out our [Comprehensive Guide to 
+Terraform](https://blog.gruntwork.io/a-comprehensive-guide-to-terraform-b3d32832baca)): 
+ 
+{% include examples/explorer.html example_id='terraform-hello-world' file_id='terraform_code' id='quick_start_page' class='wide quick-start-examples' skip_learn_more=true skip_view_on_github=true skip_tags=true %}

```

Comment:
This `include` is how we can now include real example code, including one file at a time, in our docs.